python/src/KarhunenLoeveSVDAlgorithm_doc.i.in

```diff
@@ -24,24 +24,39 @@ centeredSample : bool, default=False
     
 Notes
 -----
-The Karhunen-Loeve SVD algorithm solves the Fredholm problem  associated to the covariance function :math:`C`: see :class:`~openturns.KarhunenLoeveAlgorithm` to get the notations.
-
-The SVD approach is a particular case of the quadrature approach (see :class:`~openturns.KarhunenLoeveQuadratureAlgorithm`)  where we consider the functional basis  :math:`((\vect{\theta}_p^j(\vect{s}))_{1 \leq j \leq d, 1 \leq p \leq P}` of :math:`L^2(\cD, \Rset^d)` defined on :math:`\cD` by:
+The Karhunen-Loeve SVD algorithm solves the Fredholm problem  associated to the
+covariance function :math:`C`: see :class:`~openturns.KarhunenLoeveAlgorithm`
+to get the notations.
+If the mesh is regular, then this decomposition is equivalent to the principal
+component analysis (PCA) (see [pearson1907]_, [hotelling1933]_, [jackson1991]_, [jolliffe2002]_).
+In other fields such as computational fluid dynamics for example,
+this is called proper orthogonal decomposition (POD) (see [luo2018]_).
+
+The SVD approach is a particular case of the quadrature approach (see
+:class:`~openturns.KarhunenLoeveQuadratureAlgorithm`)  where we consider the
+functional basis  :math:`((\vect{\theta}_p^j(\vect{s}))_{1 \leq j \leq d, 1 \leq p \leq P}` of
+:math:`L^2(\cD, \Rset^d)` defined on :math:`\cD` by:
 
 .. math:: 
 
     \vect{\theta}_p^j(\vect{s})= \left[\mat{C}(\vect{s}, \vect{s}_p) \right]_{:, j}
 
-The SVD approach is used when the covariance function is not explicitly known but only through :math:`K` fields of the associated stochastic process :math:`\vect{X}`: :math:`(\vect{X}_1, \dots, \vect{X}_K)`.
+The SVD approach is used when the covariance function is not explicitly known but only
+through :math:`K` fields of the associated stochastic process :math:`\vect{X}`: :math:`(\vect{X}_1, \dots, \vect{X}_K)`.
```

Comment:
Maybe state that X is a discretized version of the stochastic process on a time grid? Currently, the reader has to guess this.

Reply:
What is $K$?